src/cpp/core/Debug.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * Debug.cpp
```

Comment:
This needs some scoping to be Windows-only, right?

Reply:
(I see the ifdefs, but should we not make the entire function Windows-only?)

Reply:
You're right -- I'll make that change.